geotags/views.py

```diff
@@ -47,16 +49,18 @@
             ORDER BY sounds_sound.created DESC
                      %(end)s"""
 
-def generate_json(sound_queryset):
-    # When using Django Sound queries:
-    #sounds_data = [[s.id, s.geotag.lat, s.geotag.lon] for s in sound_queryset]
+def generate_bytearray(sound_queryset):
+    # sounds as bytearray
+    packed_sounds = cStringIO.StringIO()
+    for sound in sound_queryset:
+        packed_sounds.write(struct.pack("i", sound.id))
+        packed_sounds.write(struct.pack("i", int(sound.lat*1000)))
+        packed_sounds.write(struct.pack("i", int(sound.lon*1000)))
 
-    sounds_data = [[s.id, s.lat, s.lon] for s in sound_queryset if not math.isnan(s.lat) and not math.isnan(s.lon)]
-
-    return HttpResponse(json.dumps(sounds_data), content_type="application/json")
+    return HttpResponse(packed_sounds.getvalue(), content_type='application/octet-stream')
```

Comment:
Can we also gzip this? I don't know the best way to make that happen. Should we do it in django, or is there something that we can tell nginx to make it compress responses to these endpoints?

```
alastair@mahler:~/code/freesound$ ls -l geotags_box*
-rw-r--r-- 1 alastair alastair 227904 Jan 12 10:46 geotags_box_barray
-rw-r--r-- 1 alastair alastair 815860 Jan 12 10:46 geotags_box_json

-rw-r--r-- 1 alastair alastair 105619 Jan 12 10:46 geotags_box_barray.gz
-rw-r--r-- 1 alastair alastair 294641 Jan 12 10:46 geotags_box_json.gz
```
Binary gets us 4x smaller than json, and gzipping another 2x smaller, which is a pretty good result, I think